frog_pictures.rb

```diff
@@ -0,0 +1,75 @@
+require 'colorize'
```

Comment:
Good work putting all this in its own file - this makes for a clear separation of concerns.